### PR TITLE
Added section for BOM reports with example

### PR DESCRIPTION
**Pull request by SergeoLacruz**

### CODE REVIEW

**Comment by matmair** [comment]
Hi there @SergeoLacruz thanks for the PR! That was fast. The maintainer is AFK for a bit - a slow response has nothing to do with this PR - which looks great.

I recon you worked with reports a bit now. If you like and have time I would be interested in your opinion regarding inventree/InvenTree/issues/2777 - if it would be useful and general insights.

**Comment by SergeoLacruz** [comment]
Hi,
Thanks for the flowers :-)
No problem. We all have other things to do. 
Michael

File: docs/report/bom.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+title: BOM Generation
+---
+
+## BOM Generation
+The bill of materials is an essential part of the documentation that needs to be send to the facctory. A sinple csv export is OK to be importet into SMT machines. But for human redable documentation it might not be suficcient. Additional information ist needed. The Inventree report system allows to generate BOM well formatted BOM reports.
```
Comment (SchrodingersGat) [inline docs/report/bom.md:6]: Spelling

- "send" - "sent"
- "facctory" - "factory"
- "sinple" - "simple"
- "importet" - "imported"
- "redable" - "readable"
- "suficcient" - "sufficient"
- "ist" - "is"

**Comment by SchrodingersGat** [comment]
Cheers @SergeoLacruz

**Pull request merged**